src/app/physics/forces/lesson/ForcesLesson.tsx

```diff
@@ -0,0 +1,688 @@
+      </p>
+      <FootballComponentsDiagram />
+      <p>Drag and drop below to derive the equations.</p>
+      <ForceComponentDerivation />
+    </Block>,
+
+    // Slide 7: Coding up how to split a force.
+    <Block color="yellow" title="Code Force Components" mode="pickcode">
+      <iframe
+        id="forceComponentsPlugin"
+        title="Code Force Components"
+        width="100%"
+        style={{ height: "calc(100vh - 100px)" }}
+        src="https://dev.pickcode.io/lesson/force-components-logic-lesson-cmcdpn8cs000ek3y1wegxmemi-2025-07-18-09-40-00"
+      ></iframe>
+    </Block>,
+
+    // Slide 8: Check in.
+    <Block color="purple" title="Check in">
+      <QuizQuestion
+        question="If you have a force of 10 N to the right, what is the upward component of that force?"
+        choices={[
+          {
+            text: "7.12 N",
+            isCorrect: false,
+            explanation: "Nope, not quite.",
+          },
+          {
+            text: "0 N",
+            isCorrect: true,
+            explanation:
+              "Exactly! Since the entire vector is to the right, it has zero going up.",
+          },
+          {
+            text: "10 N",
+            isCorrect: false,
+            explanation: "Nope, not quite.",
+          },
+        ]}
+      />
+      <QuizQuestion
+        question="If you have a force of 20 N to the up and right at a 60 degree angle above horizontal, what is the horizontal component of that force?"
+        choices={[
+          {
+            text: <ReactKatex>{`$14 \\sqrt{3}$ N`}</ReactKatex>,
+            isCorrect: false,
+            explanation: "Nope, not quite.",
+          },
+          {
+            text: "10 N",
+            isCorrect: true,
+            explanation:
+              "Exactly! We take cosine of 60 degrees to get 0.5, which we multiply by 20 N to get 10 N.",
+          },
+          {
+            text: <ReactKatex>$20 \pi$ N</ReactKatex>,
+            isCorrect: false,
+            explanation: "Nope, not quite.",
+          },
+        ]}
+      />
+    </Block>,
+
+    // Slide 9: Introducing different types of forces.
+    <Block color="yellow" title="Forces come in many types">
+      <p>
+        <Emphasize>
+          Several distinct forces occur in mechanical systems.
+        </Emphasize>{" "}
+        Understanding each allows us to model interactions accurately. Here are
+        a few examples you may see in this physics course:
+      </p>
+      <ul className="list-disc list-inside">
+        <li>
+          <KeyTerm>
+            Gravity <ReactKatex>($F_g$)</ReactKatex>
+          </KeyTerm>{" "}
+          — the universal attraction between masses, acting downward near
+          Earth's surface.
+        </li>
+        <li>
+          <KeyTerm>
+            Normal force <ReactKatex>($F_N$)</ReactKatex>
+          </KeyTerm>{" "}
+          — the force when two objects are touching each other, such as you
+          standing on the ground.
```

Comment:
hmm you don't have to fix this, but I don't love how overflowing text doesn't indent with the bullet point